sympy/sets/fancysets.py

```diff
@@ -370,16 +371,26 @@ def doit(self, **kwargs):
 
 class Range(Set):
     """
-    Represents a range of integers. Can be called as Range(stop),
-    Range(start, stop), or Range(start, stop, step); when stop is
-    not given it defaults to 1.
+    Represents a discrete range of values with uniform spacing.
+    Can be called as Range(stop), Range(start, stop), or Range(start, stop, step);
+    when stop is not given it defaults to 1.
 
     `Range(stop)` is the same as `Range(0, stop, 1)` and the stop value
     (juse as for Python ranges) is not included in the Range values.
 
-        >>> from sympy import Range
+        >>> from sympy import Range, symbols
         >>> list(Range(3))
         [0, 1, 2]
+        >>> sr, sp, st = symbols('sr, sp, st', positive=True)
```

Comment:
what about single letter symbols? Maybe they are more readable in the docstring.

Reply:
I would agree. And this is low content in showing that all values can be replaced. I would just show
```
>>> fromsympy.abc import i
>>> Range(0, i)
Range(0, i, 1)
>>> _.subs(i, 2)
Range(0, 2, 1)
```